### PR TITLE
Add new homepage design [WIP]

### PR DESCRIPTION
**Pull request by maya**
:warning: **WIP** :construction: 

This adds the new homepage design.

**Done:**
- [x] Add new written content from https://github.com/18F/writing-lab/issues/57 cc @brittag

**TODO:**
- [x] Add call to action button below each section
- [x] Add DAP column to table data
- [ ] Do we want the HTTPS donut to be bar graph instead? @konklone 

This is what it looks like:

xz8osaaaaaelftksuqmcc

### CODE REVIEW

File: static/scss/main.scss [hunk 1]
```diff
@@ -217,48 +219,129 @@ a#skipnav {
...
+          float: right;
+          margin-left: 25px;
+        }
+      }
 
-      .total-value {
-        font-size: 5em;
+      .https_desc, .dap_desc {
+        vertical-align: top;
+        padding: 27px 0 0 0;
```
Comment (maya) [inline static/scss/main.scss]: Fixed in 6a69856.

File: static/scss/main.scss [hunk 2]
```diff
@@ -217,48 +219,129 @@ a#skipnav {
...
+    h4 {
+      font-size: 2em;
+    }
+  }
+
+  #results {
+    clear: both;
+    margin: 0 auto;
+    max-width: 840px;
+    padding: 40px 40px;
```
Comment (maya) [inline static/scss/main.scss]: Fixed in 6a69856.

File: static/scss/main.scss [hunk 3]
```diff
@@ -217,48 +219,129 @@ a#skipnav {
 
 #home {
   .hero {
-    @include span-columns(8);
-    @include shift(2);
-
     text-align: center;
     color: white;
-    padding-top: 40px;
+    padding: 40px 0 60px 0;
```
Comment (maya) [inline static/scss/main.scss]: Fixed in 6a69856.

File: static/scss/main.scss [hunk 4]
```diff
@@ -217,48 +219,129 @@ a#skipnav {
...
+      .dap_chart {
+        @media (min-width: 600px) {
+          float: right;
+          margin-left: 25px;
+        }
+      }
 
-      .total-value {
-        font-size: 5em;
+      .https_desc, .dap_desc {
```
Comment (maya) [inline static/scss/main.scss]: Fixed comma sequence in 6a69856.

File: static/scss/main.scss [hunk 5]
```diff
@@ -217,48 +219,129 @@ a#skipnav {
...
+        padding-left: 0;
+      }
+    }
+
+    #main-content {
+      .tab-menu {
+        width: 100%;
+        margin-top: 40px;
+        .tab {
+          width: 33.33%;
```
Comment (maya) [inline static/scss/main.scss]: Fixed in 6a69856.

File: static/scss/main.scss [hunk 6]
```diff
@@ -217,48 +219,130 @@ a#skipnav {
...
+          float: right;
+          margin-left: 25px;
+        }
+      }
 
-      .total-value {
-        font-size: 5em;
+      .https_desc, 
+      .dap_desc {
+        vertical-align: top;
```
Comment (maya) [inline static/scss/main.scss]: Fixed in c7afc69.

File: static/scss/main.scss [hunk 7]
```diff
@@ -217,48 +219,130 @@ a#skipnav {
...
+      .dap_chart {
+        @media (min-width: 600px) {
+          float: right;
+          margin-left: 25px;
+        }
+      }
 
-      .total-value {
-        font-size: 5em;
+      .https_desc,
```
Comment (maya) [inline static/scss/main.scss]: Fixed whitespace in c7afc69.

**Comment by konklone** [comment]
> - [ ] Add call to action button below each section

:+1:

> - [ ] Add DAP column to table data

:+1:, though I continue to be _so ready_ to replace this table plugin, like, any time.

> - [ ] Do we want the HTTPS donut to be bar graph instead? @konklone

I do think it's important to find some way to show the 3 "tiers" of HTTPS support, especially as we go into 2016. One way to do this would be to use a bar chart in place of a donut for HTTPS. Another way, which might make things seem more stylistically consistent, would be to do concentric donuts for the three HTTPS %'s. Failing that, we could also change which number we promote to be one of the others, but they all have downsides associated with being the only number.

**Comment by maya** [comment]
Added call to action links in 38f248c.

screen shot 2015-12-23 at 5 00 53 pm

**Comment by konklone** [comment]
BTW, the stats for HTTPS across the whole federal .gov list on Pulse is:

* Uses HTTPS: **36%**
* Enforces HTTPS: **22%**
* Uses HSTS: **5%**

**Comment by maya** [comment]
@konklone this is what it looks like with a bar graph instead of a donut chart for HTTPS:

bars cropped

**Comment by konklone** [comment]
I'm a big fan of this bar chart, and appreciate how the color change to green makes it consistent with the donuts we use elsewhere on the site. I also appreciate that the donut works just as well on a white background as blue. I further appreciate that it makes the homepage less overwhelmingly blue. =)

This is a good step forward from our current homepage. There are other things I'd love to know your thoughts on, like a quick-search bar for domains and agency names, and I know we still need to add a column to the table below for DAP results, but this is looking good to me.

**Comment by maya** [comment]
@konklone how about something like this for the search bar? The following design removes the large "Pulse" in the main area and makes slightly larger the "Pulse" in the left corner, as well as makes the tagline more prominent.

new-design-1

cc @xtine

**Comment by maya** [comment]
I styled the search. This is what it looks like:

sa8wcnbtafbeiabeiabeiabeiabeiabeiabeiabeiabeiabeiabeiabeiabeiabeiabeiabejacwiu4hgbjbeqaamqaamqaamqaamqaamqaamqaamqaamqaamqaamqaamqaamqaamqaamqaamqaamqgj8i0aoon0aygxigarigarigarigarigarigarigarigarigarigarigarigarigarigarigariggdrj4p9fhalbfm7xswaaaabj

cc: @konklone @xtine

**Comment by konklone** [comment]
@gbinal Though this will clearly take some work to merge into the main branch, is this still something we're good to integrate? It was a lot of good work, and I think it improves the homepage substantially.

**Comment by gbinal** [comment]
It's a lot of good work and is worth returning to if we can one day but for now, I'm going to go ahead and close this.  Thanks, folks.  I'm sorry about this.

**Pull request closed without merging**